code/modules/mob/living/simple_animal/hostile/bees.dm

```diff
@@ -181,6 +186,10 @@
 	if(beehome)
 		beehome.bee_resources = min(beehome.bee_resources + growth, 100)
 
+/mob/living/simple_animal/hostile/poison/bees/proc/tray_cooldown(obj/machinery/hydroponics/H)
```

Comment:
why is this a proc on the bee and not the tray? this should be a proc on the tray